www/components/Navbar/Item.js

```diff
@@ -1,6 +1,6 @@
 import React, { PropTypes } from 'react';
 import Link from 'gatsby-link';
-import theme from '../../theme';
+// import theme from '../../theme';
```

Comment:
What's the reason to comment these out instead of removing them?

Reply:
Sorry, I merged and then the UI changed to reflect your comment. Fair call. Have created another PR to clean this up - https://github.com/keystonejs/keystone/pull/4448